config/jobs/kubernetes/publishing-bot/publishing-bot-periodics.yml

```diff
@@ -0,0 +1,87 @@
+periodics:
+- name: periodic-publishing-bot
+  cluster: k8s-infra-prow-build-trusted
+  annotations:
+    testgrid-dashboards: sig-release-publishing-bot
+    testgrid-tab-name: periodic
```

Comment:
Better to have a more specific tab name rather than the generic `periodic`.

Reply:
As suggested, updated to `publish`